### PR TITLE
add Installed-Size field

### PR DESCRIPTION
**Pull request by huakim**
modified:   main.sh

### CODE REVIEW

File: src/main.sh [hunk 1]
```diff
@@ -502,6 +502,10 @@ write_control_info() {
 	write_control_pair "Provides" "${provides[@]}"
 	write_control_pair "Replaces" "${replaces[@]}"
 	write_control_pair "Breaks" "${breaks[@]}"
+    if [ -d ${pkgdir} ] 
+    then
```
Comment (Elsie19) [inline src/main.sh]:
```suggestion
    if [[ -d ${pkgdir} ]]; then
```

File: src/main.sh [hunk 2]
```diff
@@ -502,6 +502,10 @@ write_control_info() {
...
+        write_control_pair "Installed-Size" "$(du ${pkgdir} -d 0 | cut -f1)"
+	fi
```
Comment (Elsie19) [inline src/main.sh]:
```suggestion
    fi
```

**Review by Elsie19 — CHANGES_REQUESTED** [review]

**Comment by huakim** [comment]
.

**Comment by Elsie19** [comment]
🤨

**Comment by hwittenborn** [comment]
@huakim: Is something wrong? Is it because of the long review time or something else? If it's just because I haven't gotten to reviewing this yet, I've just been terribly busy lately working on other projects that have been taking up a ton of my time.

Is it that, or is something else going on?

**Pull request closed without merging**